packages/headless/src/api/commerce/product-listings/v2/product-listing-v2-request.ts

```diff
@@ -0,0 +1,44 @@
+import {AnyFacetResponse} from '../../../../features/facets/generic/interfaces/generic-facet-response';
+import {SortCriterion} from '../../../../features/sort/sort';
+import {ProductRecommendation} from '../../../search/search/product-recommendation';
+import {
+  baseProductListingV2Request,
+  ProductListingV2RequestParam,
+} from './product-listing-v2-params';
+
+export type ProductListingV2Request = ProductListingV2RequestParam;
+
+export const buildProductListingV2Request = (req: ProductListingV2Request) => {
+  const queryStringArguments: Record<string, string> = {};
```

Comment:
If this is always an empty map, and this is the only place we call `baseProductListingV2Request`, can we remove the `queryStringArguments` argument entirely?